gulpfile.coffee

```diff
@@ -25,6 +27,10 @@ gulp.task 'coffee', ->
 		.pipe(coffee(bare: true, header: true))
 		.pipe(gulp.dest(OPTIONS.directories.build))
 
+gulp.task 'schemas', ->
+	gulp.src(OPTIONS.files.schemas, { cwd: 'extras' })
```

Comment:
Does this mean we're just copying the schemas to another directory? If so, could we put them on the final directory from the start?

Reply:
The files are copied to the `build` directory, which is not in source control.